testing/lib/workarea/test_case.rb

```diff
@@ -169,13 +169,26 @@ def reset_s3
       end
     end
 
+    module MorrisDayAndTheTime
```

Comment:
What is this? Can we stick with descriptive naming?

Reply:
I lol'd

Reply:
![image](https://user-images.githubusercontent.com/113026/69255192-628a2d00-0b85-11ea-8e11-b29af19a9797.png)

I honestly didn't know what to call it. Ran out of names yesterday.